### PR TITLE
extended LinkMapper to support external links

### PR DESCRIPTION
**Pull request by Lyra1337**
Hi,
I ran into an exception while getting linked items from our TFS2017. So I added support for external links.

Please review my changes.

### CODE REVIEW

File: src/Qwiq.Core/Proxies/ExternalLinkProxy.cs [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        {
+            get { return this.externalLink.LinkedArtifactUri; }
+        }
+
+        public string ArtifactLinkTypeName
+        {
+            get { return this.externalLink.ArtifactLinkType.Name; }
+        }
+
+        //public Tfs.RegisteredLinkType ArtifactLinkType
```
Comment (utsavized) [inline src/Qwiq.Core/Proxies/ExternalLinkProxy.cs]: Commented code can be removed.

Comment (Lyra1337) [inline src/Qwiq.Core/Proxies/ExternalLinkProxy.cs]: Yes, i forgot that. That's done now.

File: src/Qwiq.Core/Proxies/ExternalLinkProxy.cs [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        {
+            this.externalLink = externalLink;
+        }
+
+        public string LinkedArtifactUri
+        {
+            get { return this.externalLink.LinkedArtifactUri; }
+        }
+
+        public string ArtifactLinkTypeName
```
Comment (rjmurillo) [inline src/Qwiq.Core/Proxies/ExternalLinkProxy.cs]: We want to proxy the `ArtifactLinkType`

**Comment by rjmurillo** [comment]
Hi @Lyra1337, thanks for your contribution! If you have not already, please follow the link provided by the @msftclas bot to sign a license agreement so we can accept your contribution.

**Comment by msftgits** [comment]
Hi, I am closing and re-opening this PR to bump the CLA bot. Sorry for the inconvenience!

**Comment by rjmurillo** [comment]
Thank you for your contribution @Lyra1337! We have merged and deployed out to our myget feed 🎉

**Pull request merged**